Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -4114,6 +4190,157 @@ def get_cve_command(args: dict) -> list[CommandResults]:
+                    "severity": resources[0].get('severity'),
+                    "user_name": resources[0].get('user_name'),
+                    "command_line": resources[0].get('command_line'),
+                    "entity_type": resources[0].get('entity_type')
+                }
+                hr = tableToMarkdown(f'Change ID {change_id} details', returned_data)
+            else:
+                hr = "### No Change ID details found"
+
+            return CommandResults(
+                outputs_prefix='CrowdStrike.FileVantage.ChangeDetails',
+                outputs_key_field='id',
+                outputs=raw_res.get('resources')[0],
+                readable_output=hr,
+                raw_response=raw_res
+            )
+        else:
+            return "### No Change ID details found"
+    except Exception as e:
+        return str(e)
+
+
+def get_spotlight_vulnerabilities_list_command(args: dict):
+    """
+        Get a list of vulnerability IDs from Spotlight
+        :param after:   On your first request, don't provide an after token. On subsequent requests, 
+                        provide the after token from the previous response to continue from that place in the results.
+        :param limit: The maximum number of result to return.
+        :param time_range: The time when the change was reported to the CrowdStrike cloud. Define with FQL.
+    """
+    after = args.get('after')
+    limit = args.get('limit', 1000)
+    time_range = args.get('time_range', 'now-1m')
+    try:
+        raw_res = get_spotlight_vulnerabilities_list(after=None, limit=1000, time_range='now-1m')
+        if raw_res:
+            resources = raw_res.get('resources', [])
+            if resources:
+                hr = tableToMarkdown('Vulnerability IDs list', resources, headers=["Vulnerability IDs"])
+            else:
+                hr = "### No Vulnerability IDs available within the time range (default is last 1 minute)"
+            return CommandResults(
+                outputs_prefix='CrowdStrike.Spotlight.VulnerabilitiesList',
+                outputs_key_field='trace_id',
+                outputs={
+                    "trace_id": raw_res.get('meta').get('trace_id'),
+                    "total": raw_res.get('meta').get('pagination').get('total'),
+                    "resources": raw_res.get('resources')
+                },
+                readable_output=hr,
+                raw_response=raw_res
+            )
+        else:
+            return "### No Vulnerability IDs available within the time range (default is last 1 minute)"
+    except Exception as e:
+        return str(e)
+
+
+def get_spotlight_vulnerabilities_details_command(args: dict):
+    """
+        Get the full list of properties about one or more changes using the ID of each change
+
+        :param vul_id: The vulnerability ID to get details
+    """
+    vul_id = str(args.get('vul_id'))
```

Comment:
there is no argument with the name `vul_id`
```suggestion
    vul_id = str(args.get('change_id'))
```